### ISSUE TITLE
Don't work with nested `package`s

### ISSUE THREAD

**Issue by beautyfree** [zgid123/grpc-ts#1]
I get errors `Cannot find namespace` then I use nested packages

---
 I have proto files which I want to use to make grpc client which will work on fastify and communicate with other service on Rust:
 
 projectname.proto
 ```proto
syntax = "proto3";
package projectname;

import "chain_id.proto";

message SomeRequest {
  projectname.type.UserWallet user_wallet = 1; 
  optional uint32 limit = 2;
  projectname.type.ChainId chain_id = 3;
}
 ```
 
 chain_id.proto
 ```proto
syntax = "proto3";
package projectname.type;

// https://chainagnostic.org/CAIPs/caip-2
message ChainId {
  string caip2id = 1;
}
 ```
 
 and then i use `gen-grpc-typing` i get:
 
 projectname.interface.ts
 ```ts
 import type { Metadata, GrpcTimestamp, ServiceClient } from '@grpc.ts/core';

export const PACKAGE_NAME = 'projectname';

export interface SomeRequest {
  userWallet: Iprojectname.type.UserWallet;
  limit?: number;
  chainId: Iprojectname.type.ChainId;
}
```
and other files containing error `Cannot find namespace 'Iprojectname'.ts(2503)`

**Comment by zgid123** [comment]
could you share the proto folder + proto file and the config file? I will have a look

**Comment by beautyfree** [comment]
yeah, of course, i tried with files provided above and they have this structure:
```
/proto
  - chain_id.proto
  - projectname.proto
/src/protobufTypings
  - .gitkeep
grpc-cli.ts
```

grpc-cli.ts
```ts
import type { IConfigProps } from '@grpc.ts/cli';

const config: IConfigProps = {
  paths: ['./proto/**/*.proto'],
  external: ['google.protobuf'],
  output: './src/protobufTypings',
};

export default config;
```

package.json
```
{
   ...
   "scripts": {
      ...
      "gen-type": "gen-grpc-typing"
   }
}
```

and after call `pnpm gen-type` i see errors in file `src/protobufTypings/projectname.interface.ts`
<img width="510" alt="image" src="https://github.com/zgid123/grpc-ts/assets/656531/4493dc25-d029-44fd-b1eb-76eab7bb26cd">

**Comment by zgid123** [comment]
I see. When I import a proto file,  I will use the message directly `UserWallet user_wallet = 1;` instead of `projectname.type.UserWallet user_wallet = 1;`. Lemme try to handle your case. But I wonder where do you define UserWallet in your project? I need to think how to handle your case correctly

**Comment by zgid123** [comment]
I am working on this. But it will take time because at the moment, I am busy with my work. In the meantime, try to fix it manually. I will release new version asap. Thanks for the issue

**Comment by zgid123** [comment]
hi @beautyfree, I already published `1.1.0` for CLI to support the nested packages. Please let me know if that fixes your issue. If not, please gimme the full proto define, I will have a look at that

**Comment by zgid123** [comment]
hi @beautyfree, if you don't have any issue, then I will close this issue for now. Feel free to reopen if you have any problem.

Thanks for this issue.
